### ISSUE TITLE
IVsPackageInstaller2 interface install actions doesn't support floating range

### ISSUE THREAD

**Issue by erdembayar** [NuGet/Home#12355]
It's important to have floating version support for security and maintenance of .Net projects.
Currently there's no way to install floating version with [IVsPackageInstaller2](https://github.com/NuGet/NuGet.Client/blob/ba3fa10c1ae0fc4b57f4fa01c8fda0abf47f865a/src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPackageInstaller2.cs) from another VS component, only exact version.

**Comment by zivkan** [comment]
@erdembayar NuGet should not be calling its own extensibility APIs. Extensibility APIs are for other VS components only. PM UI should call whatever internal APIs that our extensibility APIs call.

Also, since this is about an extensibility API, the Functionality:SDK should be added.

**Comment by erdembayar** [comment]
> @erdembayar NuGet should not be calling its own extensibility APIs. Extensibility APIs are for other VS components only. PM UI should call whatever internal APIs that our extensibility APIs call.
> 
> Also, since this is about an extensibility API, the Functionality:SDK should be added.

Yes, this one is needed another VS extension. I reworded the description.

**Comment by jeffkl** [comment]
Team Triage: Following a response from Andy, close the issues if we are going to use a different solution.
